docs/onboarding/azure-devops-pipelines.md

```diff
@@ -55,7 +55,8 @@ If you don't wish to send usage data to Microsoft, you can set the `customerUsag
 * [Step 6 - Configure Azure Policies](#step-6---configure-azure-policies)
 * [Step 7 - Configure Hub Networking](#step-7---configure-hub-networking)
 * [Step 8 - Configure Subscription Archetypes](#step-8---configure-subscription-archetypes)
-
+* Migration
```

Comment:
Also, we should have a reference to this migration section in the "Step 3 - Management Groups" section of this document. Reading the document sequentially, that is where we want to alert the reader that "if they have an existing MG hierarchy, then they should check out this appendix for info on how to auto-generate the required YAML".